src/modules/TextOverlay/info.json

```diff
@@ -8,6 +8,10 @@
             "desc": "Enter the text to overlay.",
             "default": "Lorem ipsum"
         },
+        "Custom coordinates": {
+            "type": "coordinate-input",
+            "id": "coordinates-input"
```

Comment:
`id` is no longer required :)

Reply:
Also, please add a description `desc` if possible.

Reply:
i have added that but @jywarren wants non-bold text instead.
should i add `desc` back now?